ckanext/canada/plugins.py

```diff
@@ -44,11 +44,12 @@
 log = logging.getLogger(__name__)
 
 
-class CanadaDatasetsPlugin(SchemingDatasetsPlugin):
+class DataGCCADatasets(SchemingDatasetsPlugin):
```

Comment:
data.gc.ca was an old domain for the site, so `CanadaDatasetsPlugin` might be the better name to try to be uncoupled from future site name changes :-)

Reply:
Yay! `CanadaDatasetsPlugin` it will be!